docker/README.md

```diff
@@ -49,27 +49,30 @@ storage:
 > The `listen` directive tells Polynote to listen on all interfaces inside the Docker container. 
 > While this is probably fine for trying it out, it's not fit for production settings. 
 >
-> The `storage` directive specifies `/opt/notebooks` as the primary notebook location. It also tells Polynote to 
-> mount the examples directory that is present in the Docker distribution as a secondary mount, allowing you to see the
-> examples and try them out! 
-> Note that changes to the examples might not be persisted onto the host if you follow the instructions below.
+> The `storage` directive specifies `/opt/notebooks` as the primary notebook location. 
+> This is the location that is visible in the file-explorer on the left in Polynote. 
+> The Polynote installation comes with some example files, which we mount into `/opt/notebooks`, 
+> allowing you to see the examples and try them out without having to make copies.
+> Note that changes to the examples will not be persisted onto the host if you follow the instructions given.
+> Only files in the `notebooks` folder visible in the sidebar will persist.
```

Comment:
Hmm, you shouldn't see a `notebooks` folder if it's configured properly. Everything in the sidebar should be persisted _except_ for changes made in the `examples`.

Reply:
good catch! Left over from the documentation changes from earlier. still haven't gone through README again (as per the TODO).